docs/chart-review.md

```diff
@@ -191,36 +193,34 @@ simply pass in a new dictionary like so: `--symptoms-bsv /in/my-symptoms.bsv`.
 
 This file should look like (this is a portion of the default Covid dictionary):
 ```
-##  Columns = CUI|TUI|CODE|SAB|STR|PREF
+##  Columns = CUI|TUI||STR|PREF
```

Comment:
Whoops, should have done this in the recent PR about 4-cols, but doing it here is fine too. We gracefully handle 6-cols anyway, but let's document the intended format.